src/core/lib/debug/stats_data.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 gRPC authors.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Automatically generated by tools/codegen/core/gen_stats_data.py
```

Comment:
Thinking about this... let's do it at the same time we do copyright stuff... we should be able to put together a little module that sets up a .c/.h file as required, and also has the right hooks to do the sanity checking.